lib/index.js

```diff
@@ -115,6 +115,15 @@ module.exports = {
 
     options = merge({}, DEFAULT_OPTIONS, options);
     return runBrowsertime(url, domScript, options).then(result => {
+      if (
+        Array.isArray(result.errors) &&
+        result.errors.length !== 0 &&
+        result.errors[0].length !== 0
+      ) {
+        const browsertimeError = new Error(result.errors);
+        browsertimeError.name = 'BROWSERTIMEERROR';
```

Comment:
Can you just rename this to BrowsertimeError instead, that's how we try to name errors.